docs/src/pages/guides/FHIRSearchConfiguration.md

```diff
@@ -18,14 +18,14 @@ The IBM FHIR Server allows deployers to define search parameters on a tenant-spe
 Tenant search parameters are defined via a [Bundle](https://www.hl7.org/fhir/r4/bundle.html) of [SearchParameter](https://www.hl7.org/fhir/r4/searchparameter.html) resources that define the additional search parameters which describe the searchable field and define the FHIRPath expression for extraction.  For example, a tenant that extends the `Patient` resource type with the `favorite-color` extension, enables search on `favorite-color` by defining a SearchParameter as part of this bundle.
 
 ## 1 Configuration
-There are three layers of search parameter configuration.  
-- built-in parameters from the core specification and packaged implementation guides
-- default tenant parameters
-- tenant-specific parameters
+Search parameter definitions come from the core specification, packaged implementation guides, and other resources provided to the registry via the FHIRRegistryResourceProvider SPI.
 
-The built-in SearchParameters are loaded from the `fhir-registry` in the `fhir-search` module during server startup.
+Since IBM FHIR Server 4.10.0, *all* SearchParameters are loaded from the `fhir-registry` in the `fhir-search` module during server startup.
```

Comment:
```suggestion
Since IBM FHIR Server 4.10.0, *all* SearchParameters are loaded from the `fhir-registry` into the `fhir-search` module during server startup.
```

Reply:
used `by`